diagnostic.md

```diff
@@ -46,13 +49,14 @@ while (x < 5) {
 How many times does the loop run?  What's the value of `x` after the while loop?
 
 _Replace this text with your answer._
+4
```

Comment:
Nope, sorry. Try working this one out by hand - play the role of the computer and execute the program line by line.